lib/Ohmbrewer_Onewire.h

```diff
@@ -25,17 +26,30 @@ namespace Ohmbrewer {
          */
         double getReading();
 
+        /**
+         * outputs probe IDs and their current temperatures to the screen
+         *
+         * Primarily for use as a manual identification tool for probe IDs
+         */
+        void displayProbeIds(Screen *screen);
+
+        /**
+         * assigns the probe ID to supplied onewire probe
+         * @param probe id  Unique probe ID
+         * @param probe  - probe
+         */
+        void assignProbeId(Ohmbrewer::Onewire probe, uint8_t (&probeId)[8]);//TODO
```

Comment:
It looks like you want this to be a static method. You're trying to make something that isn't associated with a particular instance of the class, right?

Otherwise, I don't see a whole lot of advantage of this over the method it's delegating to (setProbeId).